[AGM-2062] Update AdGuard for Android features description (#494)
AdguardTeam/KnowledgeBase: docs/adguard-for-android/solving-problems/low-level-settings.md

If this setting is enabled, AdGuard will test DNS upstreams before adding or updating custom DNS servers.

#### Filter secure DNS

> If this setting is on, AdGuard will apply all enabled DNS filters and DNS user rules to encrypted DNS-over-HTTPS (DoH) traffic, and not just to plain DNS. DoH traffic mostly comes from Chrome and other browsers that have a “Secure DNS” setting (or similar). You can use the “Filter secure DNS” feature in two different modes:

Reviewer: ```suggestion
If this setting is enabled, AdGuard will apply all enabled DNS filters and DNS user rules to encrypted DNS-over-HTTPS (DoH) traffic and not only to plain DNS. DoH traffic mostly comes from Chrome and other browsers that have a “Secure DNS” (or similar) setting. You can use the “Filter secure DNS” in two different modes:
```
